microcosm_flask/swagger/schemas.py

```diff
@@ -86,6 +92,11 @@ def iter_schemas(self, schema: Schema) -> Iterable[Tuple[str, Any]]:
         if not schema:
             return
 
+        if type(schema) in self.seen_schemas:
```

Comment:
I think this is ok.  We'll never have a schema instantiated twice and mean different things, right?  cc/ @dumpstate @afallou @GabeLambda

Reply:
If you did, that would seem like a PEBCAK issue.